api/server.go

```diff
@@ -0,0 +1,40 @@
+package api
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/rs/zerolog/log"
+	"github.com/sprintertech/sprinter-signing/api/handlers"
+)
+
+func Serve(
+	ctx context.Context,
+	addr string,
+	signingHandler *handlers.SigningHandler,
+) {
+	mux := http.NewServeMux()
+	mux.HandleFunc("POST /signing", signingHandler.HandleSigning)
+
+	server := &http.Server{
```

Comment:
you should set read timeouts (to prevent slowloris attacks) and definitely enable https://github.com/securego/gosec linter.